docs/getting-started/endgame-sensor-FDA-ven.asciidoc

```diff
@@ -0,0 +1,110 @@
+[[endgame-sensor-full-disk-access-ven]]
+= Enable Full Disk Access for the Endgame sensor on macOS Ventura and higher
+
+:frontmatter-description: Manually install and deploy Elastic Endgame on macOS Ventura and higher.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [secure] 
+
+If your solution includes Elastic Endgame, it requires Full Disk Access to protect you from malware and other cybersecurity threats. Full Disk Access permissions is a privacy feature that controls which applications can access your data. This means you need to manually enable Full Disk Access permission for the Endgame sensor to access these protected areas of your Mac.
+
+[discrete]
+[[macos-permissions-ven]]
+== macOS permissions
+
+The behavior of the Endgame sensor differs based on your macOS version. MDM/Jamf users can pre-approve all Full Disk Access without granting permission to the sensor. However, depending on the macOS version and sensor type, non-MDM/Jamf users may be prompted to enable Full Disk Access for required security files.
+
+Here are the following Full Disk Access requirements for specific versions of macOS:
+
+- `10.15`: You cannot install the sensor without allowing it to load a <<kernel-ext-approval-endgame, kernel extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, installation proceeds.
+
+- `11.0`, `12.0`: You cannot install the sensor without allowing it to load a <<system-extension, system extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, a second prompt appears to enable Network Filtering. Approve this final prompt to proceed.
++
+You must also grant Full Disk Access to `com.endgame.systemextension`.
+
+- `10.15, 11.0, 12.0`: Grant the <<endpoint-endgame-sensor, esensor file>> Full Disk Access.
+
+NOTE: The following instructions apply to the Endgame sensor only. To see requirements for the {elastic-endpoint}, refer to <<deploy-elastic-endpoint>>.
+
+[discrete]
+[[kernel-ext-approval-endgame-ven]]
+== Approve the kernel extension for the Endgame sensor
+
+For endpoints running macOS 10.15, the Endgame sensor will attempt to load a kernel extension (as opposed to a system extension) during installation. This kernel extension is required to provide insight into system events such as process events, file system events, and network events. The following prompt appears during installation:
+
+--
+image::images/fda/sys-ext-blocked.png[System extension blocked]
```

Comment:
Leaving a note for myself that I might need to replace this with a Ventura-specific screenshot.

cc: @Trinity2019

Reply:
Thanks for checking this, @ricardoungureanu! Just to make sure I'm understanding you, are you saying that users only need to approve the Endgame sensor kernel extension if they're on _**macOS Catalina though Monterey**_? If that's the case, I'll need to remove this section (lines [29](https://github.com/elastic/security-docs/pull/3565/files#diff-dcaa61fd73dd362c8a03ec95c8585b730ea161d9bea3de8ccd94887b5c73f0c7R29)-[57](https://github.com/elastic/security-docs/pull/3565/files#diff-dcaa61fd73dd362c8a03ec95c8585b730ea161d9bea3de8ccd94887b5c73f0c7R57)) and the reference to this section (line [19](https://github.com/elastic/security-docs/pull/3565/files#diff-dcaa61fd73dd362c8a03ec95c8585b730ea161d9bea3de8ccd94887b5c73f0c7R19)) from the `endgame-sensor-FDA-ven.asciidoc` file. (The `endgame-sensor-FDA-ven.asciidoc` file describes how to give FDA to the Endgame sensor if you're on macOs Ventura.)

Reply:
Yes, we can remove it the `Approve the kernel extension for the Endgame sensor` section from the `Ventura or higher Page`

Reply:
Resolved in [f29d871](https://github.com/elastic/security-docs/pull/3565/commits/f29d8710fcb5ee6c14bf0677d51168d581022e57).